website/use-cases.md

```diff
@@ -10,12 +10,76 @@ meta_description: What are the use cases that the CDS Registration specification
+These communications can include protocols defined by industry specification (e.g. OpenADR, IEEE 1547, etc.) or using proprietary or custom Application Programming Interface (API) protocols (e.g. Oracle, SAP, Mulesoft, etc.).
 
-See [Issue #1](https://github.com/lfe-cds/CDS-Registration/issues/1) for our progress on finalizing our use cases.
+These secure, confidential connections frequently depend on initially exchanging configuration details between the utility's server and the end client user's systems.
+However, currently these configuration details exchanges (including sharing secrets and credentials with client users) are primarily done manually using unspecified ad-hoc delivery methods (e.g. secure email), which significantly limits the capacity for utilities and other central entities to establish connectivity with external entities.
+This initial manual configuration bottleneck effectively caps the adoption of grid programs and communications that rely on external connectivity.
 
+Therefore, to enable and unblock scalable adoption of grid communication specifications and protocols, this working group must write open, freely available general connectivity specifications that provide other specifications, vendors, utilities, and other central entities with protocols that provide automated methods for initially establishing secure connectivity with external third party clients.
+
+#### Needs:
+
+* **Structured Disclosure of Capabilities:**
+Because the energy grid is highly fragmented, different utilities and other central entities offer widely varying sets of communication protocols to external entities and customers, and differentiating which utilities and other central entities offer what types of connectivity is a highly time-intensive manual task.
+These specifications must prescribe protocols by which external entities and customers may automatically discover and parse which capabilities utilities and other central entities offer.
+
+* **Automated Client Registration:**
+Because many grid communication specifications and protocols require establishing a secure, confidential connection between the utility systems ("Servers") and external systems ("Clients"), these specifications must prescribe protocols by which Servers can receive requests by Clients to register and initially exchange connection details (e.g. client credentials, endpoint URLs, etc.).
+
+#### Examples of users in this use case:
+
+* Other specifications and protocols that require secure, confidential connectivity ([CDSC Customer Data](https://customerdata.carbondataspec.org/), [CDSC Power Systems Data](https://powersystemsdata.carbondataspec.org/), [LF Energy TROLIE](https://lfenergy.org/projects/trolie/), [Super Advanced Meter](https://github.com/super-advanced-meter), [OpenADR](https://www.openadr.org/), [IEEE 1547](https://ieeexplore.ieee.org/document/8332112), etc.)
```

Comment:
Update CDSC --> CDS

Reply:
Fixed in https://github.com/lfe-cds/CDS-Registration/pull/6/commits/de6022d666cb9050d8a60b6e9d7e3fc5dd371f1e